modules/teleport-cluster/rbac-config.yaml

```
@@ -1,17 +1,17 @@
 kind: role
 version: v5
 metadata:
-  name: k8s-admin
+  name: k8s-dev-admin
```

```suggestion
  name: k8s-developer-admin
```

not to confuse with `dev` environment.